<?xml version="1.0"?>
<robot name="boat">

  <material name="white">
    <color rgba="0.25 0.25 0.25 1"/>
  </material>

  <link name="uuv">
    <visual>
    	<origin rpy="1.5708 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://uuv_simulation/meshes/VTEC_U-II_GAMMA.STL" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="white"/>
    </visual>

    <!--collision>
    	<origin rpy="1.57075 0 1.57075" xyz="0 0 -0.25"/>
      <geometry>
        <mesh filename="package://urdf_tutorial/meshes/boat.dae" scale="0.001 0.001 0.001"/>
      </geometry>
    </collision-->

    <!--inertial>
      <origin rpy="1.57075 3.1416 1.57075" xyz="0 0 -0.25"/>
      <mass value="30"/>
      <inertia
        ixx="1.0" ixy="0.0" ixz="0.0"
        iyy="1.0" iyz="0.0"
        izz="4.1"/>
    </inertial-->
  </link>
</robot>